.gitignore

```diff
@@ -5,3 +5,7 @@ __pycache__
 .pytest_cache
 .tox
 MANIFEST
+
+# IDE
+.idea
+.idea/*
```

Comment:
i use `pycharm` for python work, just thought its good to have these in the `.gitignore` to avoid `idea` folder from getting checked in